raddb/mods-available/rest

```diff
@@ -19,22 +19,22 @@ rest {
 #		ca_path = ${certdir}
 
 #		certificate_file        = /path/to/radius.crt
-#		private_key_file	= /path/to/radius.key
-#		private_key_password	= "supersecret"
-#		random_file		= /dev/urandom
+#		private_key_file        = /path/to/radius.key
```

Comment:
We would really prefer patches which don't randomly change whitespace.

Reply:
I think it's ok, there some messed up tab/space stuff that this fixes.